packages/block-library/src/table/edit.js

```diff
@@ -161,22 +167,25 @@ function TableEdit( {
 	 *
 	 * @param {Array} content A RichText content value.
 	 */
-	function onChange( content ) {
-		if ( ! selectedCell ) {
-			return;
-		}
-
-		setAttributes(
-			updateSelectedCell(
-				attributes,
-				selectedCell,
-				( cellAttributes ) => ( {
-					...cellAttributes,
-					content,
-				} )
-			)
-		);
-	}
+	const onChange = useCallback(
+		function ( content ) {
+			if ( ! selectedCell ) {
+				return;
+			}
+
+			setAttributes(
+				updateSelectedCell(
+					attributes,
+					selectedCell,
+					( cellAttributes ) => ( {
+						...cellAttributes,
+						content,
+					} )
+				)
+			);
+		},
+		[ attributes, selectedCell, setAttributes ]
```

Comment:
We can use the updater function here, which eliminates the need to pass attributes as dependencies. Should make memoization more "stable". Ref: #69709.